### PR TITLE
PHP 8 support

### LINKED ISSUE #4: PHP 8.0 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

To be prepared for the december release of PHP 8.0, this repository has some additional TODOs to be tested against the new major version.

In order to make this repository compatible, one has to follow these steps:

- [ ] Modify `composer.json` to provide support for PHP 8.0 by adding the constraint `~8.0.0`
- [ ] Modify `composer.json` to drop support for PHP less than 7.3
- [ ] Modify `composer.json` to implement phpunit 9.3 which supports PHP 7.3+
- [ ] Modify `.travis.yml` to ignore platform requirements when installing composer dependencies (simply add `--ignore-platform-reqs` to `COMPOSER_ARGS` env variable)
- [ ] Modify `.travis.yml` to add PHP 8.0 to the matrix (*NOTE*: Do not allow failures as PHP 8.0 has a feature freeze since 2020-08-04!)
- [ ] Modify source code in case there are incompatibilities with PHP 8.0

### PR DESCRIPTION
**Pull request by tanerkay**
Q 	A
New Feature 	yes

### Description

Addresses #4

### CODE REVIEW

**Comment by sivaschenko** [comment]
Hi @tanerkay `laminas/laminas-json:3.2.0` is now compatible with PHP 8

File: .travis.yml [hunk 1]
```diff
@@ -12,38 +12,28 @@ env:
...
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: 8.0
       env:
         - DEPS=lowest
-    - php: 7.3
+        - COMPOSER_ARGS="--ignore-platform-reqs"
```
Comment (froschdesign) [inline .travis.yml]: Please remove this line, it should work without ignoring requirements.

File: .travis.yml [hunk 2]
```diff
@@ -12,38 +12,28 @@ env:
...
+    - php: 8.0
       env:
         - DEPS=latest
+        - COMPOSER_ARGS="--ignore-platform-reqs"
```
Comment (froschdesign) [inline .travis.yml]: The same here.

**Comment by tanerkay** [comment]
thanks @froschdesign. PR updated

**Comment by froschdesign** [comment]
Travis is gone and the [new workflow for continuous integration](https://github.com/laminas/laminas-continuous-integration-action) must be added. (See also on other components for help, e.g. [laminas-feed](https://github.com/laminas/laminas-feed/pull/35).)

**Comment by weierophinney** [comment]
Closing in favor of #8 (which includes all commits you have provided) due to issues with GHA and the branch name.

**Pull request closed without merging**